packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -309,12 +320,12 @@ def _create_items(self, token_ids: Union[List[int], int]) -> TransactionMessage:
                 token_id=token_ids,
             )
 
-    def _mint_objects(self, is_batch: bool, address: Address, token_id: int = None):
+    def _mint_objects(self, is_batch: bool, address: Address):
```

Comment:
Since we are keeping track of the game currency in the behaviour no need to pass it to the function anymore.